transactions/btc.utils.ts

```diff
@@ -200,3 +201,10 @@ export function selectOptimalUtxos({
 
   return bestSelectionData;
 }
+
+// get ordinals utxos in btc address
+export async function getOrdinalsUtxos(btcAddress: string) {
+  const ordinals = await getOrdinalsByAddress(btcAddress);
+  const filteredOrdinals = ordinals.filter((item) => item.id !== undefined);
```

Comment:
This filter happens within `getOrdinalsByAddress`. I think it can be removed here to avoid confusion.